src/node_contextify.h

```diff
@@ -139,23 +140,23 @@ class ContextifyContext : public BaseObject {
   std::unique_ptr<v8::MicrotaskQueue> microtask_queue_;
 };
 
-class ContextifyScript : public BaseObject {
+class ContextifyScript final : public cppgc::GarbageCollected<ContextifyScript>,
+                               public cppgc::NameProvider,
+                               public CppgcMixin {
```

Comment:
nit: to reduce boilerplate, perhaps these can be put behind a macro ? Something like `class ContextifyScript final : public CPPGC_MIXIN {}` or `CPPGC_MIXIN(ContextifyScript) { ... }`

Reply:
Done